packages/google-drive-kit/src/board-server/operations.ts

```diff
@@ -38,6 +38,20 @@ export type GraphInfo = {
 
 let alreadyWarnedAboutMissingPublicApiKey = false;
 
+/** Retries fetch() calls until status is not an internal server error. */
+async function retriableFetch(
+  input: string|Request,
+  init?: RequestInit,
+  numRetries = 3
+): Promise<Response> {
+  let response: Response;
+  do {
+    response = await fetch(input, init);
+    numRetries--;
```

Comment:
We should probably wait for some time between each attempt. Ideally we'd do exponential backoff, but a fixed 1 or 2 seconds would be OK for now.

Something like this:

`await new Promise((resolve) => setTimeout(resolve, RETRY_MS))`

Reply:
done with simple delay of 200 millis since drive API isn't particularly fast, adding another 1-2 seconds delay may be too much. WDYT?